FlightReservation/src/schemas/logInSchema.js

```diff
@@ -0,0 +1,6 @@
+import * as yup from 'yup';
+
+export const logInValidationSchema = yup.object().shape({
+    email: yup.string().required('Email Address is Required'),
+    password: yup.string().required('Password is Required'),
+})
```

Comment:
It'd be better if we make this validation in the backend rather than in the client side 😄 
WDYT?

Reply:
They are practicing the basic use of firebase, don't have custom APIs, so their backend is limited to that, in fact, I consider that validation on both sides is necessary